docs/geedocs/docsrc/BUILD_DOCS.md

```diff
@@ -6,14 +6,12 @@ source files and then generate HTML using Sphinx.
 
 ## Prerequisites
 
-To build the documentation, you need to install Sphinx:
+To build the documentation, the following packages (and versions) need to be installed :
 
-```bash
-sudo apt install python-pip
-sudo pip install sphinx==1.7.5
-sudo pip install GitPython
-sudo pip install sphinxprettysearchresults
-```
+python 2.7
+git 1.8+
+python-pip
+python packages: sphinx(1.7.5), GitPython(1.0.1), sphinxprettysearchresults(0.3.5)
```

Comment:
On [the rendered md](https://github.com/google/earthenterprise/blob/91ad0c5a6bf383928f4a8dd4caed1a62f7ebcb40/docs/geedocs/docsrc/BUILD_DOCS.md) the listed packages appear on same line. If u want them to be listed in separate line, may want to add an empty line i believe.